README.md

```diff
@@ -1,2 +1,190 @@
 # dpnt-video
-Datapoint - video
+
+Merge videos uploaded by user
+
+- [Hmmm](./container/images/hmmm/)
+
+### Updating sub-modules
+
+Root project contains three git submodules:
+
+- local-sqs
+- local-s3
+- local-ecs
+
+Run the below command in the project root to update the above submodules:
+
+```bash
+git submodule update --init
+```
+
+## Acceptance test
+
+Start the local S3 and SQS simulators
+```bash
+python local-sqs/elasticmq-wrapper.py start
+python local-s3/minio-wrapper.py start
+minio config host add myminio http://192.168.1.190:9000 local_test_access_key local_test_secret_key
+```
+
+We use the `hmmm` language to test the application.
+The language container image needs to be build and tagged as `latest`:
```

Comment:
There will be only one container. No need for multiple, language specific, containers.